query_plan.go

```diff
@@ -240,12 +241,22 @@ func renderTreeWithStats(tree treeprint.Tree, linkType string, node *Node) {
 	str := "\t" + string(b)
 
 	if len(node.Children) > 0 {
-		branch := tree.AddBranch(str)
+		var branch treeprint.Tree
+		if node.IsRoot() {
+			tree.SetValue(str)
```

Comment:
I have confirmed named root is supported 👍 
https://github.com/xlab/treeprint/blob/10e0bc383e01918fbf58bfd402ddd63b3e2e08f4/treeprint_test.go#L65-L80